--- src/sys.c	2017-03-04 20:34:08.475900545 +0900
+++ src/sys.c	2017-03-04 20:32:28.000000000 +0900
@@ -13,7 +13,11 @@
 #ifdef _OS_WINDOWS_
 #include <psapi.h>
 #else
+#include <unistd.h>
+#if !defined(_SC_NPROCESSORS_ONLN) || defined(_OS_FREEBSD_) || defined(_OS_DARWIN_)
+// try secondary location for _SC_NPROCESSORS_ONLN, or for HW_AVAILCPU on BSDs
 #include <sys/sysctl.h>
+#endif
 #include <sys/wait.h>
 #include <sys/ptrace.h>
 #include <unistd.h>
