--- core_plugin.py	2019-11-11 17:46:48.000000000 +0000
+++ core_plugin.py	2019-12-17 04:09:49.910876000 +0000
@@ -332,7 +332,6 @@
     parser.add_argument(
         '--bind_all',
         action='store_true',
-        default=True,
         help='''\
 Serve on all public interfaces. This will expose your TensorBoard instance to
 the network on both IPv4 and IPv6 (where available). Mutually exclusive with
